ci-operator/step-registry/ipi/conf/vsphere/enable-template/ipi-conf-vsphere-enable-template-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: ipi-conf-vsphere-enable-template
+  from: upi-installer
+  commands: ipi-conf-vsphere-enable-template-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
```

Comment:
looks like those credentials are not used in the step

Reply:
will delete.